### ISSUE TITLE
prepInputs not working with .rar objects

### ISSUE THREAD

**Issue by tati-micheletti** [PredictiveEcology/reproducible#62]
In the reprex below, Raster1 works fine, as Raster2 fails. I thought I had corrected this issue, but apparently not. Will take a look at it ASAP.

```r
Raster1 <- prepInputs(archive = "http://biogeo.ucdavis.edu/data/diva/adm/CAN_adm.zip",
                      destinationPath = tempdir())

Raster2 <- prepInputs(url = "http://www.cec.org/sites/default/files/Atlas/Files/2010nalcms30m/can_landcover_2010_30m.rar",
                 destinationPath = tempdir())
```

**Comment by tati-micheletti** [comment]
Fixed with #86

**Comment by tati-micheletti** [comment]
FIxed with PR #86
